packages/browser/src/client/dialog.ts

```diff
@@ -0,0 +1,21 @@
+const showPopupWarning = <T>(name: string, value: T, defaultValue?: T) => (...params: any[]) => {
+  const formatedParams = params.map(p => JSON.stringify(p)).join(', ')
+
+  console.warn(`Vitest encountered a \`${name}(${formatedParams})\` call that it cannot handle by default, so it returned \`${value}\`. Read more in https://vitest.dev/guide/browser#thread-blocking-dialogs.
```

Comment:
@sheremet-va You also need to update the tests for the new text! I can do it but I'm busy with the playwright integration right now.